core/controllers/acl_decorators.py

```diff
@@ -2506,6 +2506,17 @@ def test_can_accept_suggestion(
                     self.user_id, suggestion.score_category):
                 return handler(self, target_id, suggestion_id, **kwargs)
 
+            if suggestion.suggestion_type == (
```

Comment:
>> Perhaps. As soon as a reviewer leaves comments on your code, it means that there is at least one reader to whom the distinction is not clear. So I would suggest answering the question within the code itself, for the benefit of other readers of the code in the future. Adopt this as a general practice.
>>
>> More specifically: you explained in a reply to the original comment that the condition will never be true and that the whole scoring structure will be removed in a future PR. Both of these points individually seem like important information that future readers of the code won't know anything about, and so it should be documented (unless I misinterpreted your comment above).

> Yeah, these conditions are True but I don't think a reader has to know this to understand the new changes OR the new changes are not made considering "these" conditions i.e, it will work without considering the above condition.

I understand that it will work. But I think documentation i still important -- I'm not thinking about the new changes, I'm thinking about the method logic as a whole after this PR gets merged.

You've mentioned that check_user_can_review_in_category will never be true as we don't store the user's score, and that you plan to remove the whole existing score structure in one PR. None of these things are obvious from reading the code and so they should be documented. In general comments should explain the "why" of things, and the current open "why" question is "why is this dead code sitting here". The other open question is why we appear to have have two distinct checks for "user can review", which seems confusing.

Reply:
Done!